pkgs/by-name/me/mealie/mealie-logs-to-stdout.patch

```diff
@@ -0,0 +1,103 @@
+diff --git a/mealie/core/root_logger.py b/mealie/core/root_logger.py
+index 29db504f..3da5ef84 100644
+--- a/mealie/core/root_logger.py
++++ b/mealie/core/root_logger.py
+@@ -9,7 +9,6 @@ DATA_DIR = determine_data_dir()
+ 
+ from .config import get_app_settings  # noqa E402
+ 
+-LOGGER_FILE = DATA_DIR.joinpath("mealie.log")
```

Comment:
Would it be enough to replace the file here with /dev/stdout or /dev/log?

Reply:
This was confirmed to not work previously,